gba_cart.ld

```diff
@@ -160,8 +160,8 @@ SECTIONS
 	.iwram __iwram_start : AT (__iwram_lma)
 	{
 		__iwram_start__ = ABSOLUTE(.) ;
-		*(.iwram .iwram*)
-		*iwram.*(.text* .data*)
+		*(.iwram)
```

Comment:
Could you change this to include `.iwram.*` instead of `.iwram*` as before?
Otherwise, linking with libseven will break, which uses `.iwram.{FunctionName}` names.